docs/local-ui-dev.md

```diff
@@ -0,0 +1,85 @@
+# Local UI Development
+
+This directory contains tools and instructions for using the RE api in a local development context which allows close operation with other kbase services in a custom docker network.
```

Comment:
there are a couple of different ways of referring to the RE API in the docs -- can you standardise them all to be `RE API`?

Reply:
done